service/elastigroup/providers/aws/aws.go

```diff
@@ -917,6 +918,14 @@ type StopDeploymentInput struct {
 
 type StopDeploymentOutput struct{}
 
+type MetadataOptions struct {
+	HttpTokens              *string `json:"httpTokens,omitempty"`
```

Comment:
```suggestion
	HTTPTokens              *string `json:"httpTokens,omitempty"`
```

The [convention in Go](https://golang.org/doc/effective_go.html#mixed-caps) is to use `MixedCaps` (for public variables) or `mixedCaps` (for private variables) to write multiword names. Words that are acronyms (like HTTP or URL) are [written in full upper case](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).